test/integration/shoots/maintenance/maintenance.go

```diff
@@ -0,0 +1,338 @@
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
+	"github.com/gardener/gardener/test/framework"
+	integrationframework "github.com/gardener/gardener/test/integration/framework"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+var (
+	testMachineryRun      = flag.Bool("test-machinery-run", false, "indicates whether the test is being executed by the test machinery or locally")
+	testShootCloudProfile gardencorev1beta1.CloudProfile
+
+	// Test Machine Image
+	highestShootMachineImage gardencorev1beta1.ShootMachineImage
+	testMachineImageVersion  = "0.0.1-beta"
+	testMachineImage         = gardencorev1beta1.ShootMachineImage{
+		Version: &testMachineImageVersion,
+	}
+
+	// Test Kubernetes versions
+	testKubernetesVersionLowPatchLowMinor             = gardencorev1beta1.ExpirableVersion{Version: "0.0.1", Classification: &deprecatedClassification}
+	testKubernetesVersionHighestPatchLowMinor         = gardencorev1beta1.ExpirableVersion{Version: "0.0.5", Classification: &deprecatedClassification}
+	testKubernetesVersionLowPatchConsecutiveMinor     = gardencorev1beta1.ExpirableVersion{Version: "0.1.1", Classification: &deprecatedClassification}
+	testKubernetesVersionHighestPatchConsecutiveMinor = gardencorev1beta1.ExpirableVersion{Version: "0.1.5", Classification: &deprecatedClassification}
+
+	// cleanup
+	gardenerSchedulerReplicaCount *int32
+	baseShoot                     gardencorev1beta1.Shoot
+	shootCleanupNeeded            bool
+	cloudProfileCleanupNeeded     bool
+
+	// other
+	setupContextTimeout      = time.Minute * 2
+	restoreCtxTimeout        = time.Minute * 2
+	deprecatedClassification = gardencorev1beta1.ClassificationDeprecated
+	expirationDateInThePast  = metav1.Time{Time: time.Now().UTC().AddDate(0, 0, -1)}
+	err                      error
+	f                        *framework.ShootCreationFramework
+)
+
+func init() {
+	framework.RegisterShootCreationFrameworkFlags()
+}
+
+const (
+	waitForCreateDeleteTimeout = 7200 * time.Second
+	initializationTimeout      = 600 * time.Second
+)
+
+var _ = ginkgo.Describe("Shoot Maintenance testing", func() {
+	f = framework.NewShootCreationFramework(&framework.ShootCreationConfig{
+		GardenerConfig: &framework.GardenerConfig{
+			CommonConfig: &framework.CommonConfig{
+				ResourceDir: "../../../framework/resources",
+			},
+		},
+	})
+
+	framework.CBeforeSuite(func(ctx context.Context) {
+		if testMachineryRun != nil && *testMachineryRun {
+			f.Logger.Info("Running in test Machinery")
+			// setup the integration test environment by manipulation the Gardener Components (namespace garden) in the garden cluster
+			// scale down the gardener-scheduler to 0 replicas
+			replicas, err := integrationframework.ScaleGardenerScheduler(setupContextTimeout, f.GardenClient.Client(), pointer.Int32Ptr(0))
```

Comment:
Same question here...

Reply:
This is required because this integration test creates a Shoot before and cleans it after each test. Having a scheduled shoot is not desired here because it would be picket up by the Gardenlet / start reconciliing -> slows down the tests and incurs costs.

Reply:
Sure, I got that this is required. I just wanted to asked, if this test is even run and if so against which system...

Reply:
@schrodit this should be executed against the regular landscape correct?  So this would actually scale down the scheduler in each landscape? How is that currently handled and is that acceptable (the test already exist in a previous version since a while)?